main.tex

```diff
@@ -295,11 +296,11 @@ \section*{Abstract} %(301 out of 300 words)
 \section*{Author summary}
 
 We revise a provenance ontology for simulation studies of cellular biochemical models.
-Provenance information is useful for understanding the creation of a simulation model, because it does not only contain information about the entities and activities that have led to a simulation model but also the relations of these, which can be visualized.
-It provides additional structure as research questions, assumptions, and requirements are singled out and explicitly related along with data, qualitative models, simulation models, and simulation experiments through a small set of predefined but extensible activities.
+Provenance information is useful for understanding the creation of a simulation model because it not only contains information about the entities and activities that have led to a simulation model, but also the relations, all of which can be visualized.
```

Comment:
Just reworded for clarity

Reply:
I added "their" (-> "also their relations") ok or not? (And if the comma before "but" is not mandatory, I would leave it out.)
Provenance information is useful for understanding the creation of a simulation model because it not only contains information about the entities and activities that have led to a simulation model but also their relations, all of which can be visualized.

Reply:
Oh yeah that comma can go and yes "their" is a good change!